tests/README.md

```diff
@@ -173,11 +176,13 @@ Example:
     5. `tox -e py -- tests/unit` (runs the unit tests under the default python interpreter with `recommenders[all]`)
     6. `tox -e py37 -- tests/unit` (runs the unit tests under Python3.7 with `recommenders[all]` )
 
-## How to create tests on notebooks with Papermill
+## How to create tests
+
+### How to create tests on notebooks with Papermill
```

Comment:
Here we need to add also scrapbook. In the past Papermill allowed for executing and collecting metrics in the notebooks, however, since its version 1.0, we use papermill for parametrizing and executing the notebooks, and scrapbook for getting the metrics